style/footer.scss

```diff
@@ -0,0 +1,93 @@
+
+
+  footer{
+    background-color: #56bc94;
+    color:white;
+    background-image: url("pattern-1.png");
+    padding-bottom:15px;
+  }
+  .footer-text{
+  flex-direction: column;
+  align-items: center;
+    display: flex;
+    padding-top: 50px;
+     p{
+        text-align: center;
+       
+        width: 45%;
+      }
+  }
+ 
+  .information{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+        input , textarea{  /*error check this out*/
+        padding:15px;
+        border:transparent;
+        border-radius: .25rem;
+        margin-bottom: 15px;
+        width: 40%;
+      }
+  }
+ 
+  .btn-massege{
+  width: 15%;;
+  margin-top: 15px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 15px;
+  
+  
+    button{
+        margin-left: auto;
+        width: 100%;
+        border: 1px solid #56bc94;;
+        color:white;
+        font-size:25px;
+        border-radius: .25rem;
+    
+        background-color: #317b5e;
```

Comment:
since you're using scss let's define some variables for these colors (not just for reusability but also for readability/self-documenting-code)